src/farmers/farmerbot/farmerbot_information.md

```diff
@@ -216,33 +222,50 @@ This ensures an optimal user experience and reliablity in 3Nodes' reports.
 
 Note that each node managed by the Farmerbot will randomly wakeup on average 10 times a month.
 
-
-
 ### Farmerbot and UTC time
 
 The Farmerbot works in UTC, independently of the local time of your machine.
 
+### Upgrade the Old Farmerbot to the New Farmerbot
 
+If you are still running the old version of the Farmerbot (written in V), you can easily upgrade to the new Farmerbot (written in Go). You simply need to properly stop the old Farmerbot and then follow the new [Farmerbot guide](./farmerbot_quick.md).
 
-## Maintenance
+Here are the steps to properly stop the old Farmerbot.
 
-### See the Farmerbot logs
+* Go to the diretory with the old Farmerbot docker files and fully stop the old Farmerbot:
+  ```
+  docker compose rm -f -s -v
+  ```
+* You should also make sure that there are no containers left from the previous runs. First, list all containers:
+  ```
+  docker container ls --all
+  ```
+* Then delete the remaining containers:
+  ```
+  docker container rm -f -v NAME_OF_CONTAINER
+  ```
 
-* Go to your farmerbot directory
-* Go to config folder
-  * ```
-    cd config
-    ```
-* Check logs
-  * ```
-    cat farmerbot.log
-    ```
+Once the old Farmerbot is properly stopped and deleted, follow the new [Farmerbot guide](./farmerbot_quick.md).
 
+### Set the Farmerbot without the mnemonics with a twin associated with a polkadot.js extension account
 
+If your twin is associated with a polkadot.js extension account, but you do not have the mnemonics, you can set the Farmerbot with certain conditions.
 
-### See the power state and power target of a 3Node with GraphQL
+This is possible if your exported JSON lists `ed25519` under encoding. If it lists `sr25519`, it is not possible. 
 
-On [GraphQL](https://graphql.grid.tf/graphql), write the following lines, with the proper farm ID (here we set **1**):
+For `ed25519`, you will need to extract the hex secret.
+
+That being said, if you do not have your mnemonics for your polkadot.js extension account, the simplest way forward is to create a new polkadot.js account and generate a new twin. Then you will need to create a new farm and migrate the existing nodes to the new twin and farm. This is actually very simple: when you boot your new bootstrap image of the new farm, your node will move to the new farm, with its current node ID. 
+
+If you are using the Farmerbot, at this point, you will be able to set it with the mnemonics associated with the new farm.
+
+## Maintenance
+
+### See the power state and power target of 3Nodes with GraphQL
+
+You can use [GraphQL](https://graphql.grid.tf/graphql) to see the power state and power target of 3Nodes. 
+
```

Comment:
It's also possible through the Gird proxy, like https://gridproxy.grid.tf/nodes/655/.

In the response, there would be 
"power": {
    "state": "string",
    "target": "string"
  },

Reply:
Done. Gave an example with an activate status bot too (node id 1)